ops/terraform/aws/network/main.tf

```diff
@@ -0,0 +1,204 @@
+provider "aws" {
+  region = "${var.region}"
+}
+
+# VPC
+resource "aws_vpc" "application_vpc" {
+  cidr_block           = "${var.vpc_cidr_block}"
+  instance_tenancy     = "dedicated"
```

Comment:
I think we're ok with shared tenancy for Soapbox. Maybe a particular user could request dedicated, one that is particularly paranoid about hypervisor security. But I think dedicated tenancy is more expensive and prevents you from using the smallest instance sizes (maybe I'm wrong).